src/vic3save/src/savefile.rs

```diff
@@ -0,0 +1,268 @@
+use crate::stats::Vic3CountryStats;
+use crate::Vic3Date;
+use serde::de;
+use serde::de::Unexpected;
+use serde::{de::DeserializeOwned, Deserialize, Deserializer};
+use std::collections::HashMap;
+use std::fmt;
+use std::hash::Hash;
+use std::marker::PhantomData;
+
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct MetaData {
+    pub version: String,
+    pub game_date: Vic3Date,
+}
+
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct Counters {
+    week: i32,
+    tick: i32,
+}
+
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct Vic3CountryBudget {
+    pub weekly_income: Vec<f32>,
+}
+
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct PopStatistics {
+    pub trend_population: Vic3CountryStats,
+}
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct Vic3Country {
+    pub definition: String,
```

Comment:
I haven't seen one different than 3 charcaters. Dynamic tags are `D01`, `D02`, .... idk what happens if that goes over 100.

Are you suggesting to use that struct? or copy it? It would probably work

Reply:
Up to you, I was curious as I don't know the inner workings of Vic3. If we can copy and paste the same country tag between implementations that would be nice, but don't sweat it too much.

Reply:
I don't want to replace it right now